scripts/security-checker.mjs

```diff
@@ -137,38 +178,43 @@ class SecurityChecker {
               description:  alert.security_advisory.description,
               link:         alert.html_url,
               issuePackage: alert.dependency.package.name,
+              cveId:        alert.security_advisory.cve_id,
+              ghsaId:       alert.security_advisory.ghsa_id,
           });
       }
   }
 
   async createCodeqlIssues (codeqlAlerts) {
       for (const alert of codeqlAlerts) {
-          if (!this.needCreateIssue(alert))
-              return;
+          if (!this.needCreateIssue(alert, false))
+              continue;
 
           await this.createIssue({
               labels:      [LABELS.codeql, LABELS.security],
               originRepo:  this.context.repo,
               summary:     alert.rule.description,
               description: alert.most_recent_instance.message.text,
               link:        alert.html_url,
-          });
+          }, false);
       }
   }
 
-  needCreateIssue (alert) {
-      return !this.alertDictionary[alert.html_url] && Date.now() - new Date(alert.created_at) <= 1000 * 60 * 60 * 24;
+  needCreateIssue (alert, isDependabotAlert = true) {
+      const alert = isDependabotAlert ? alert.security_advisory.summary : `[${this.context.repo}] ${alert.rule.description}`;
```

Comment:
It's not an alert itself, it's a key for the dictionary. Moreover, you already have this name in the scope of this method